server/GRPC_TA2_TA3/relay.js

```diff
@@ -0,0 +1,58 @@
+// Javascript Imports
+const path = require("path");
+const grpc = require("grpc");
+const WebSocketServer = require("ws").Server;
+
+// Environment variables for GRPC
+let GRPC_PORT = process.env.GRPC_PORT
+let GRPC_HOST = process.env.GRPC_HOST
+let GRPC_CONN_URL = GRPC_HOST+ ":" +GRPC_PORT
+
+// Websocket defs
+let wss = new WebSocketServer({port: 8888});
+
+// Loading proto files
+let core = grpc.load(path.join(__dirname, "ta3ta2_api", "core.proto"), 'proto', {enumsAsStrings:false});
```

Comment:
Please add a space between colon and "false": ``{enumsAsStrings: false}``.
These lines are over-lengthy. Let's try to keep max line length no larger than 80.